src/sparsezoo/model/utils.py

```diff
@@ -99,10 +156,12 @@ def load_files_from_stub(
     )
     # piece of code required for backwards compatibility
     files = restructure_request_json(response_json["model"]["files"])
+    compressed_file_size = _get_compressed_size(response_json["model"]["files"])
     model_id = response_json["model"]["model_id"]
     if params:
         files = filter_files(files, params)
-    return files, model_id, params
+    validation_results = _parse_validation_metrics(response_json["model"]["results"])
```

Comment:
What happens in the case that there are no validation results/ results are missing? is `["model"]["results"]` guaranteed to exist with an empty list?

Reply:
+1, we are being too reliant on those files actually being there.

Reply:
Addressed, instead of direct access, I changed to use `.get(...)`